silhouette/test/com/mohiva/play/silhouette/api/SilhouetteSpec.scala

```diff
@@ -834,7 +834,7 @@ class SilhouetteSpec extends PlaySpecification with Mockito with JsonMatchers {
         }
       }))
 
-      env.eventBus.subscribe(listener, ct.runtimeClass)
+      env.eventBus.subscribe(listener, ct.runtimeClass.asInstanceOf[Class[T]])
```

Comment:
because the Classifier now need a `Class[_ <: SilhouetteEvent]` and runtimeClass return Class[_]